x-pack/plugins/security_solution/common/api/endpoint/suggestions/get_suggestions.gen.ts

```diff
@@ -22,9 +22,7 @@ export type GetEndpointSuggestionsRequestParams = z.infer<
   typeof GetEndpointSuggestionsRequestParams
 >;
 export const GetEndpointSuggestionsRequestParams = z.object({
-  query: z.object({
-    suggestion_type: z.literal('eventFilters').optional(),
-  }),
+  suggestion_type: z.literal('eventFilters'),
```

Comment:
I'm not really familiar with the use of this, thus I don't know if making it required is going to be an issue. 
I would like to have @tomsonpl look at it first since he seemed to have been the original author of this file.

Reply:
_(commenting on behalf of the Defend Workflows team)_

Reply:
I am not sure If I understand why this interface changes ? And why just this one ? @xcrzx 

Previously we had:
```
{
   query: {
      suggestion_type?: string;
   }
} 
```
and now it's just:
```
{
      suggestion_type: string;
} 
```

Reply:
The actual change is in the API's OpenAPI schema.

<img width="1351" alt="image" src="https://github.com/elastic/kibana/assets/1938181/a109977d-4b25-49aa-90cb-2b5e465b6c8b">

The spec defines a path parameter named `query` that doesn't exist. It also described it as an object with the attribute `suggestion_type`. However, path parameters can't be objects since only a string can be part of a URL. It seems this was just an oversight in the spec, so I've corrected it.

@tomsonpl Let me know if you have any questions on OpenAPI spec usage, we can have a quick chat on that offline